docs/spec.openapi.yml

```diff
@@ -70,12 +70,18 @@ paths:
             Object:
               schema:
                 $ref: '#/components/schemas/ResSession'
+        '401':
+          description: Unauthorized
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResSession'
         'default':
           description: An error occurred, see other codes at [Authentication API Spec `POST /challenge`](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
           content:
             Object:
               schema:
-                $ref: '#/components/schemas/ResStatus'
+                $ref: '#/components/schemas/ResSession'
```

Comment:
Technically we always return a ResSession but it's not guaranteed that the relevants fields will be filled if there is an expetion (i.e. a code differenet from 20x).

So instead let is as ResStatus which is guaranteed to be complete (exception), also it allows us to be consistent with the other endpoints. Which always return a ResStatus if something wrong happens.